kubernetes/assets/monitors/monitor_statefulset_replicas.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "[kubernetes] Monitor Kubernetes Statefulset Replicas",
 	"type": "query alert",
-	"query": "avg(last_15m):avg:kubernetes_state.statefulset.replicas_desired{*} by {statefulset} - avg:kubernetes_state.statefulset.replicas_ready{*} by {statefulset} >= 2",
+	"query": "max(last_15m):sum:kubernetes_state.statefulset.replicas_desired{*} by {statefulset} - sum:kubernetes_state.statefulset.replicas_ready{*} by {statefulset} >= 2",
 	"message": "More than one Statefulset Replica's pods are down.",
```

Comment:
Usually this presents an unsafe situation for any further manual operations like killing other pods. It maybe worthwhile adding that to the message here but that sounds very prescriptive than just saying something is wrong. Not sure we want to do that here.

Reply:
I'll add it to the message - I'd rather err on the side of being a bit more opinionated.